src/components/SubscriptAvatar.js

```diff
@@ -34,6 +37,7 @@ const defaultProps = {
     mainAvatar: {},
     secondaryAvatar: {},
     noMargin: false,
+    showTooltip: true,
 };
 
 function SubscriptAvatar(props) {
```

Comment:
Here I only put the components in variables so we can conditionally render the tooltip and avoid duplicating code